src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -118,6 +118,11 @@
             <Paragraph>By default, this tool window is docked with the
                 <Italic>Solution Explorer</Italic>. However, you can move it any where you like, just like any other tool window.
             </Paragraph>
+            <Paragraph>
```

Comment:
Please lmk if this is the correct link. Also, I have made the whole sentence clickable but I can also change it so only the "here" is.

Reply:
Yes, the link is correct.

I would have included the url in the text, to make it more obvious that clicking the link would take the user out of VS e.g. `You can browse all available rules at https://rules.sonarsource.com.`

Let's see what Marco thinks next week.